doc: mention errors thrown by methods called on an unbound dgram.Socket (#33983)
nodejs/node: doc/api/dgram.md

one interface and will add membership to it. To add membership to every
available interface, call `addMembership` multiple times, once per interface.

> When called on an unbound socket, implicitly binds to a random port, listening
> on all interfaces.

Reviewer: This kind of sentence fragment (subject is unstated) is used in the first sentence in an entry, but subsequently, we identify the subject.

```suggestion
When called on an unbound socket, it will implicitly bind to a random port,
listening on all interfaces.
```

membership to it. To add membership to every available interface, call
`socket.addSourceSpecificMembership()` multiple times, once per interface.

> 

Reviewer: ```suggestion
```

> When called on an unbound socket, implicitly binds to a random port, listening

Reviewer: Same comment as above (adding subject).

For UDP sockets, this object will contain `address`, `family` and `port`
properties.

> This method throws `EBADF` if called on an unbound socket.

Reviewer 2: ```suggestion
This method throws an `Error` with code `EBADF` if called on an unbound socket.
```

Author: I think these should be wrapped into an appropriate Node error anyway. Leaving as is for now.

* Returns: {number} the `SO_RCVBUF` socket receive buffer size in bytes.

> Throws [`ERR_SOCKET_BUFFER_SIZE`][] if called on an unbound socket.

Reviewer: This also doesn't have a subject but we make lots of exceptions for `Throws` throughout the docs, I think, so I think this and the subsequent ones are OK.

Author: Added subject to all the sentences. Thanks for the review!
